lib/mix/tasks/appsignal.diagnose.ex

```diff
@@ -410,7 +410,7 @@ defmodule Mix.Tasks.Appsignal.Diagnose do
 
       {:error, %{reason: reason}} ->
         IO.puts("  Error: Something went wrong while submitting the report to AppSignal.")
-        IO.puts(reason)
+        IO.puts(inspect(reason))
```

Comment:
```suggestion
        IO.inspect(reason)
```

Reply:
Actually, looking at [`IO.inspect`'s docs](https://hexdocs.pm/elixir/IO.html#inspect/2), it pretty-prints -- not sure if that's what we want, as it could make the error message take a lot of space in the terminal?

Reply:
🤷‍♂️ Could help the customer figure out what's wrong if it's easier to read, but if the error is very long it may also obscure our message above it.